src/main/java/core/basesyntax/service/impl/CsvFileReportWriterService.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.ReportRecord;
+import core.basesyntax.service.ReportWriterService;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.List;
+
+public class CsvFileReportWriterService implements ReportWriterService {
+    private static final String HEADER = "fruit,quantity" + System.lineSeparator();
+    private final String filename;
+
+    public CsvFileReportWriterService(String filename) {
+        this.filename = filename;
+    }
```

Comment:
same as for reader:
1. complicated specific name for class
2. filename should be method argument